tests/mock_logger.cpp

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2020 Canonical, Ltd.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; version 3.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "mock_logger.h"
+
+namespace mp = multipass;
+namespace mpl = multipass::logging;
+namespace mpt = multipass::test;
+using namespace testing;
+
+mpt::MockLogger::MockLogger(const PrivatePass&)
+{
+}
+
+auto mpt::MockLogger::inject() -> Scope
```

Comment:
Same question as above...

Reply:
The advantage is clearer here, the alternative would be: `mpt:MockLogger::Scope mpt::MockLogger::inject()`.

Reply:
Yep, I see.  Nice!